localisation/00_wc_sad.csv

```diff
@@ -704,6 +725,7 @@ whispers_option_stewardship;
 whispers_option_intrigue;�Y[Root.GetFirstName]�! will focus on writing down whispers about �YIntrigue�!.;;;;;;;;;;;;;x
 whispers_option_learning;�Y[Root.GetFirstName]�! will focus on writing down whispers about �YLearning�!.;;;;;;;;;;;;;x
 nick_the_corruptor;the Corruptor;;[From.Getderdie] Verderber[From.GetinFemale];;;;;;;;;;;x
+nick_twilight_of_the_gods;Twilight of the Gods;;G�tterd�mmerung;;;;;;;;;;;x
```

Comment:
I'm not an expert, but shouldn't it be?
```suggestion
nick_twilight_of_the_gods;the Twilight of the Gods;;G�tterd�mmerung;;;;;;;;;;;x
```

Reply:
its an easter egg based on the german loc being the name of final part of wagners opera

Reply:
Well, Götterdämmerung is even older, from the norse Ragnarök in the Saga...

Reply:
> its an easter egg based on the german loc being the name of final part of wagners opera

The before the eastern egg won't hurt it.

Reply:
Generally speaking, I'd say you're right, zumbak, but just like nicknames like "Lionheart", this does not explicitly need "the" before it, I think.

Reply:
I prefer to follow CK2 style:
```nick_the_lionheart;the Lionheart;Coeur de Lion;Lцwenherz;;El Corazуn de Leуn;;;;;;;;;x```

Reply:
@zumbak04 You're right, I just quick-read two articles about it and checked historic use of the lionheart. In German and French they didn't use article for it, but English did (and still do)... So, "the" should be back on the table :D